cloud/jenkins/openshift-cluster-destroy.yml

```
@@ -0,0 +1,60 @@
+- job:
+    name: openshift-cluster-destroy
+    project-type: pipeline
+    description: |
+        Destroy OpenShift clusters on AWS.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)